main.go

```diff
@@ -1,22 +1,47 @@
 package main
 
 import (
+	"errors"
 	"log"
 	"net/mail"
 	"os"
 	"strings"
-	"errors"
 
-	skprconfig "github.com/skpr/go-config"
-	extensionsv1beta1 "github.com/skpr/operator/pkg/apis/extensions/v1beta1"
+	"gopkg.in/alecthomas/kingpin.v2"
 
+	skprconfig "github.com/skpr/go-config"
 	"github.com/skpr/mail/internal/provider/mailhog"
 	"github.com/skpr/mail/internal/provider/ses"
+	extensionsv1beta1 "github.com/skpr/operator/pkg/apis/extensions/v1beta1"
+)
+
+var (
+	to                []string
+	from              string
+	recipientsFromMsg bool
+	ignoreDots        bool
 )
 
 func main() {
+
+	app := kingpin.New("skprmail", "A sendmail replacement for the Skpr hosting platform.")
```

Comment:
This might be a neater way to handle setting the vars.

https://github.com/skpr/cluster-healthz/blob/master/cmd/cluster-healthz/main.go#L16